src/context/WalletProvider/DemoWallet/DemoMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import { EditIcon } from '@chakra-ui/icons'
+import { MenuDivider, MenuItem } from '@chakra-ui/react'
+import { useCallback } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useFeatureFlag } from 'hooks/useFeatureFlag/useFeatureFlag'
+import { useWallet } from 'hooks/useWallet/useWallet'
+
+import { WalletActions } from '../actions'
+
+const editIcon = <EditIcon />
+
+export const DemoMenu = () => {
+  const translate = useTranslate()
+  const { dispatch: walletDispatch } = useWallet()
+  const isAccountManagementEnabled = useFeatureFlag('AccountManagement')
+
+  const handleManageAccountsMenuItemClick = useCallback(() => {
+    walletDispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })
+  }, [walletDispatch])
+
+  return (
+    <>
+      <MenuDivider />
+      {isAccountManagementEnabled && (
```

Comment:
do we still want the menu divider if account management is disabled? maybe we just reutnr null in that case?

Reply:
Indeed. Will switch branch context shortly and fix 🙏

Reply:
Addressed in https://github.com/shapeshift/web/pull/6724/commits/c4465086dcea348fd46ac6fb003fe49a6ceffb0d, good catch.